runtime/browser/runtime.cc

```diff
@@ -301,4 +301,13 @@ void Runtime::RequestMediaAccessPermission(
       web_contents, request, callback);
 }
 
+bool Runtime::CheckMediaAccessPermission(
```

Comment:
Isn't it better to return `false` if you just want to avoid the warnings then? Otherwise there's no check for the security origin and all checks will just pass.